src/main/java/org/spongepowered/api/world/Location.java

```diff
@@ -638,6 +631,7 @@ public Entity createEntity(EntityType type) {
         return this.getExtent().createEntity(type, this.getPosition());
     }
 
+    // TODO change this cause advisory
```

Comment:
Is this still TODO? I guess it refers to the Javadoc comment of the method below?